grossmont/computer-science-and-cybersecurity-organization/index.html

```diff
@@ -19,16 +19,17 @@
 
 <body style="margin: 0">
   <div class="fixed-header">
-    <iframe src="sections/header/header.html" width="100%" height="100%" frameborder="0"></iframe>
+    <iframe id="header" src="./sections/header/header.html" width="100%" height="100%" frameborder="0"></iframe>
```

Comment:
I added id of header and footer to the iframe which loads the respective html files, specifically so theme.js can select them.